server/src/main.ts

```diff
@@ -14,7 +14,7 @@ async function bootstrap(configService: ConfigService) {
     // app.useWebSocketAdapter(new ExtendedIoAdapter(app, corsConfig));
     app.enableCors(corsConfig)
 
-    app.useGlobalPipes(new ValidationPipe())
+    app.useGlobalPipes(new ValidationPipe({ transform: true }))
```

Comment:
Lets investigate further what this actually does, and assess again if we need that

Reply:
Turns out we don't need that at all.